libraries/botframework-connector/tests/auth.test.js

```diff
@@ -15,6 +16,7 @@ describe('Bot Framework Connector - Auth Tests', function() {
                     throw new Error('Expected validation to fail.');
                 } catch (err) {
                     assert(err.message === `'authHeader' required.`, `unexpected error thrown: "${ err.message }"`);
+                    assert.equal(err.statusCode, 400);
```

Comment:
use bad request constant?

Reply:
@Zerryth there are additional tests in this PR (in `botbuilder`) that should also use the constant.

Reply:
wowza yeah there were a lot of numbers in botbuilder tests that can now be replaced with more readable `StatusCode` enum 

updated them to do so